system/jlib/jlzw.ipp

```diff
@@ -56,6 +56,9 @@ public:
     virtual void commitblock() override;
     virtual bool adjustLimit(size32_t newLimit) override;
     virtual CompressionMethod getCompressionMethod() const override { return COMPRESS_METHOD_LZW; }
+    virtual bool supportsBlockCompression() const override { return false; }
+    virtual bool supportsIncrementalCompression() const override { return true; }
+    virtual size32_t compressBlock(size32_t destSize, void * dest, size32_t srcSize, const void * src) override { return 0; }
```

Comment:
Should this be a throwUnexpected() ?

Reply:
It could do.  0 indicates failure so I think that is also fine.